pkg/forwarder/transactions_serializer.go

```diff
@@ -0,0 +1,139 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package forwarder
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	proto "github.com/golang/protobuf/proto"
+)
+
+const transactionsSerializerVersion = 1
+
+// TransactionsSerializer serializes Transaction instances.
+// To support a new Transaction implementation, add a new
+// method `func (s *TransactionsSerializer) Add(transaction NEW_TYPE) error {`
+type TransactionsSerializer struct {
+	collection HttpTransactionProtoCollection
+}
+
+// NewTransactionsSerializer creates a new instance of TransactionsSerializer
+func NewTransactionsSerializer() *TransactionsSerializer {
```

Comment:
Same here for private ant the other methods in this file

Reply:
I think this one should stay public as it is part of Transaction interface.